manager/manifests/grafana/grafana-dashboard-async.yaml

```diff
@@ -61,10 +61,6 @@ data:
         {
           "collapsed": false,
           "datasource": null,
-          "fieldConfig": {
-            "defaults": {},
-            "overrides": []
-          },
```

Comment:
This was automatically deleted when I exported the json, should it be reverted, or is that ok?